public/locales/fr/videocall.json

```diff
@@ -19,5 +19,18 @@
             "denied": "L'accés à la caméra et au microphone a été bloqué.",
             "waiting": "Nous avons besoin d'accéder à la caméra et au microphone pour que votre correspondant puisse vous voir et vous entendre."
         }
+    },
+    "errors": {
+        "title": "Une erreur est survenue",
+        "desc": "Une erreur est survenue qui vous empêchera probablement de voir votre proche, voici le détail:",
+        "errors": {
+            "error-1655267250": "L'appel que vous avez rejoins a expiré (après 2 heures).",
+            "error--1494481250": "L'appel que vous avez rejoins n'est plus disponible (après 2 heures).",
+            "error-1189215202": "L'accés internet semble être indisponible ou pourrit.",
+            "error-763909797": "Une erreur inconnue est survenue",
+            "error--140619965": "L'appel est véroullé (trop de personne l'on rejoins ou il a expiré, en vrai j'en sais rien)",
```

Comment:
it shouldn't be `camera-error-621623973` `error-140619965` `camera-error--1412228987` ?

Reply:
no, the hashcode method return negative number too.